src/components/FlowRunResults.vue

```diff
@@ -56,26 +56,21 @@
 </template>
 
 <script lang="ts" setup>
-  import { ButtonGroupOption } from '@prefecthq/prefect-design'
   import { useSubscription } from '@prefecthq/vue-compositions'
   import { computed } from 'vue'
   import ArtifactCard from '@/components/ArtifactCard.vue'
+  import RowGridLayoutList from '@/components/RowGridLayoutList.vue'
+  import ViewModeButtonGroup from '@/components/ViewModeButtonGroup.vue'
   import { useWorkspaceApi } from '@/compositions'
   import { localization } from '@/localization'
-  import { FlowRun } from '@/models'
+  import { FlowRun, ResultArtifact } from '@/models'
```

Comment:
```suggestion
  import { FlowRun } from '@/models'
```

Reply:
love it - idk why i thought `type` was a reserved word lol